service/service.go

```diff
@@ -856,25 +841,6 @@ func (s *Service) signTx(token string, w http.ResponseWriter, r *http.Request, _
 
 	txHash, err := s.nodeForward.SendTx(r.Context(), tx, ty, cltIdx)
 	if err != nil {
```

Comment:
All the errors cases above `return` after calling `s.writeInternalError()` so I don't think we ever get to `s.policy.Report()` them?

Reply:
Good catch !